paper/paper.md

```diff
@@ -32,82 +32,125 @@ affiliations:
-chunk-wise and queried against the genome interval tree in parallel. Results are stored in a shared memory 
-object. We then convert variant hits to [Ensemble](https://uswest.ensembl.org/index.html) translation IDs, 
-in acordance to the [HGVS](https://varnomen.hgvs.org/) sequence variant nomenclature. The output is a 
-heirarchically ordered text file (csv or json) that reports the the Ensemble protein ID and the gene associated with 
-each variant, for each experimental sample. 
+associated with each observed variant. [OBB: Add a `head` output of an example vcf file]
+
+Current methods for variant calling are incredibly powerful and robust, however, a single sequencing run can generate on the order of 10^8 unique vcf entries, only a small portion of which are of relevance to the researcher. 
+In addition, variant callers report only the variant itself, and not the functional consequences of each mutation, *i.e.* the effect the mutation has on the translated protein sequence, termed "peptide-level variants." 
+We introduce *cerebra*, a python package that provides fast and accurate peptide-level summarizing of vcf files.
+
+## Functionality
+
+`cerebra` comprises three modules: (1) `germline-filter` removes variants that are common between germline samples 
+and samples of interest, (2) `count-mutations` reports total number of variants in each sample, and (3) `find-aa-mutations` reports likely peptide-level variants in each sample. 
+While `cerebra` focuses on *variants* defined specifically as single nucleotide polymorphisms (SNPs) and short insertions and deletions, other genomic variations exists, suc as larger structural variants such as copy number variations and chromosomal rearrangements. 
+
+A cruical data structure in `cerebra` is the *genome interval tree*, a data structure that matches RNA transcripts
+and peptides to each feature in the genome (*Figure 1*). 
+Interval trees are self-balancing binary search trees that store numeric intervals and can quickly find every such interval that overlaps a given query interval. 
+They have theoretical average-case O(log*n*) and worst-case O(*n*) time complexity for search operations, making them tractable for genome-scale operations. [todo: do i need a source for this? OBB: Yes] 
+Tree construction proceeds at O(*n*log*n*) time complexity, making construction rather than search the bottleneck for small vcf sets. [todo: say something about memory footprint]
+The genome interval tree is constructed with a reference genome sequence ([FASTA format](https://en.wikipedia.org/wiki/FASTA_format), often with a `.fa` extension), a genome annotation 
+([gene transfer format, GTF](https://www.gencodegenes.org/pages/data_format.html)`.gtf` extension) and a reference transcriptome (OBB: also `.fa` format? Why is this necessary? Can't the transcriptome fasta be extracted from the gtf + fasta? see: https://github.com/nf-core/rnaseq/blob/3b6df9bd104927298fcdf69e97cca7ff1f80527c/main.nf#L802). 

```

Comment:
looking into this

Reply:
ahh your totally right. not sure what i was thinking with `reference transcriptome`. probably a relic of an earlier version of the code